logging/cluster-logging-release-notes.adoc

```diff
@@ -8,20 +8,68 @@ toc::[]
 * link:https://access.redhat.com/errata/RHBA-2021:2112[RHBA-2021:2112 - Bug Fix Advisory. OpenShift Logging Bug Fix Release 5.1.0]
 
 [id="openshift-logging-supported-versions"]
 == Supported versions
 
-* OpenShift Logging version 5.1 runs on {product-title} versions 4.7 and 4.8.
+* OpenShift Logging versions 5.0, 5.1 and 5.2 run on {product-title} versions 4.7 and 4.8.
 
 [id="openshift-logging-inclusive-language"]
 == Making open source more inclusive
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
 
 // Release Notes by version
+
+[id="cluster-logging-release-notes-5-2-0"]
+== OpenShift Logging 5.2.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-TBD[RHSA-TBD OpenShift Logging Bug Fix Release 5.2.0].
+
+[id="openshift-logging-5-2-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this update, if you use the Fluentd forward protocol to forward log data over a TLS-encrypted connection, you can now use a password-encrypted private key file and specify the passphrase in the Cluster Log Forwarder configuration. For more information, see xref:../logging/cluster-logging-external.html#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]. (link:https://issues.redhat.com/browse/LOG-1525[LOG-1525])
+
+* This enhancement enables you to use a username and password to authenticate a log forwarding connection to an external Elasticsearch instance. For example, if you cannot use mutual TLS (mTLS) because a third-party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password. For more information see xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-es_cluster-logging-external[Forwarding logs to an external Elasticsearch instance]. (link:https://issues.redhat.com/browse/LOG-1022[LOG-1022])
+
+* By default, the data model introduced in OpenShift 4.5 gave logs from different namespaces a single index in common. This change made it harder to see which namespaces produced the most logs. The current release, OpenShift Logging 5.2, adds namespace metrics to the *Logging* dashboard in the {product-title} console. With these metrics, you can see which namespaces produce logs and how many logs each namespace produces for a given timestamp. To see these metrics, open the {product-title} console and click *Monitoring* -> *Logging*.
```

Comment:
I'd use the variable for {product-title} instead of OpenShift 4.5 here 🙂 unless it should be OpenShift Logging?

Reply:
Updated. Thanks.

Reply:
@pmoogi-redhat Is this the release note about "new metrics" you mentioned?

Reply:
@rolfedh  it does n't capture the name of the new metrics that we added. We can add the below to this above description. 

 "The current release, OpenShift Logging 5.2, enables two new metrics which account for total logs produced (or logged by individual containers) and total logs collected by the collector. These metrics are labeled by namespace, pod and container names so you can see how many logs produced, collected in each namespace, in each pod for a given timestamp, duration"

Reply:
Thanks @pmoogi-redhat. I revised the proposed text and added it to the release note. Please review and comment or approve.